### ISSUE TITLE
ArrayIndexOutOfBoundsException

### ISSUE THREAD

**Issue by marcellorvalle** [veraPDF/veraPDF-library#1188]
I am experiencing an issue similar to #701 

```
out 07, 2021 1:58:50 PM org.verapdf.processor.ProcessorImpl validate
WARNING: Exception caught when validating item
org.verapdf.core.ValidationException: Caught unexpected runtime exception during validation
        at org.verapdf.pdfa.validation.validators.BaseValidator.validate(BaseValidator.java:95)
        at org.verapdf.processor.ProcessorImpl.validate(ProcessorImpl.java:219)
        at org.verapdf.processor.ProcessorImpl.process(ProcessorImpl.java:120)
        at org.verapdf.processor.BatchFileProcessor.processItem(BatchFileProcessor.java:98)
        at org.verapdf.processor.BatchFileProcessor.processList(BatchFileProcessor.java:74)
        at org.verapdf.processor.AbstractBatchProcessor.process(AbstractBatchProcessor.java:102)
        at org.verapdf.gui.ValidateWorker.doInBackground(ValidateWorker.java:128)
        at org.verapdf.gui.ValidateWorker.doInBackground(ValidateWorker.java:62)
        at java.desktop/javax.swing.SwingWorker$1.call(SwingWorker.java:304)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.desktop/javax.swing.SwingWorker.run(SwingWorker.java:343)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:835)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -11 out of bounds for length 0
        at org.verapdf.pd.font.cff.CFFIndex.get(CFFIndex.java:54)
        at org.verapdf.pd.font.cff.Type2CharStringParser.execSubr(Type2CharStringParser.java:122)
        at org.verapdf.pd.font.cff.Type2CharStringParser.processNextOperator(Type2CharStringParser.java:97)
        at org.verapdf.pd.font.type1.BaseCharStringParser.parse(BaseCharStringParser.java:131)
        at org.verapdf.pd.font.type1.BaseCharStringParser.<init>(BaseCharStringParser.java:79)
        at org.verapdf.pd.font.cff.Type2CharStringParser.<init>(Type2CharStringParser.java:52)
        at org.verapdf.pd.font.cff.CharStringsWidths.getWidthFromCharstring(CharStringsWidths.java:166)
        at org.verapdf.pd.font.cff.CharStringsWidths.parseSubsetWidths(CharStringsWidths.java:249)
        at org.verapdf.pd.font.cff.CharStringsWidths.<init>(CharStringsWidths.java:95)
        at org.verapdf.pd.font.cff.CharStringsWidths.<init>(CharStringsWidths.java:118)
        at org.verapdf.pd.font.cff.CFFType1FontProgram.readWidths(CFFType1FontProgram.java:219)
        at org.verapdf.pd.font.cff.CFFType1FontProgram.parseFont(CFFType1FontProgram.java:96)
        at org.verapdf.pd.font.cff.CFFFontProgram.parseFont(CFFFontProgram.java:90)
        at org.verapdf.gf.model.impl.pd.font.GFPDCIDFont.<init>(GFPDCIDFont.java:75)
        at org.verapdf.gf.model.impl.pd.font.GFPDType0Font.getDescendantFont(GFPDType0Font.java:95)
        at org.verapdf.gf.model.impl.pd.font.GFPDType0Font.<init>(GFPDType0Font.java:57)
        at org.verapdf.gf.model.factory.fonts.FontFactory.parseFont(FontFactory.java:74)
        at org.verapdf.gf.model.impl.operator.textshow.GFOpTextShow.parseFont(GFOpTextShow.java:219)
        at
 org.verapdf.gf.model.impl.operator.textshow.GFOpTextShow.getVeraModelFont(GFOpTextShow.java:155)
        at org.verapdf.gf.model.factory.operators.OperatorParser.addFontAndColorSpace(OperatorParser.java:620)
        at org.verapdf.gf.model.factory.operators.OperatorParser.parseOperator(OperatorParser.java:305)
        at org.verapdf.gf.model.factory.operators.OperatorFactory.operatorsFromTokens(OperatorFactory.java:110)
        at org.verapdf.gf.model.impl.pd.GFPDContentStream.parseOperators(GFPDContentStream.java:148)
        at org.verapdf.gf.model.impl.pd.GFPDContentStream.isContainsTransparency(GFPDContentStream.java:177)
        at org.verapdf.gf.model.impl.pd.GFPDPage.parseContentStream(GFPDPage.java:212)
        at org.verapdf.gf.model.impl.pd.GFPDPage.getContentStream(GFPDPage.java:193)
        at org.verapdf.gf.model.impl.pd.GFPDPage.getLinkedObjects(GFPDPage.java:118)
        at org.verapdf.pdfa.validation.validators.BaseValidator.addAllLinkedObjects(BaseValidator.java:199)
        at org.verapdf.pdfa.validation.validators.BaseValidator.checkNext(BaseValidator.java:166)
        at org.verapdf.pdfa.validation.validators.BaseValidator.validate(BaseValidator.java:117)
        at org.verapdf.pdfa.validation.validators.BaseValidator.validate(BaseValidator.java:93)
        ... 13 more
```

[Document that  caused the problem](https://github.com/veraPDF/veraPDF-library/files/7305681/offending.pdf)

**Comment by MaximPlusov** [comment]
This issue duplicate issue [1163](https://github.com/veraPDF/veraPDF-library/issues/1163). Fixed in the latest dev build 1.19.205

**Comment by bdoubrov** [comment]
Fix released in v1.20
